2.Arrays/arrays.js

```diff
@@ -0,0 +1,102 @@
+console.log(
+  Object.prototype.toString.call([3,'da',4]) === '[object Array]');
+console.log([48] instanceof Array);
+
+function isArray2(arr) {
+  return Array.isArray(arr);
```

Comment:
this should be removed. and you should have 2 functions with return instanceof and Object.prototype.toString.call